test-fixtures/urn/v11/input.json

```diff
@@ -0,0 +1,21 @@
+{
+  "leader": "01331cam a22003494i 4500",
+  "fields": [
+    { "tag": "001", "value": "000000001" },
+    { "tag": "020", "ind1": " ", "ind2": " ", "subfields": [ {"code": "a", "value": "978-951-9155-47-0"} ]},
+    { "tag": "337", "ind1": " ", "ind2": " ", "subfields": [ {"code": "b", "value": "c"}, {"code": "2", "value": "rdamedia"} ] },
+    { "tag": "856", "ind1": "4", "ind2": "0", "subfields": [
+      {"code": "u", "value": "http://urn.fi/URN:ISBN:978-951-9155-47-0"},
+      {"code": "5", "value": "FI-Vapaa"}
+    ]},
+    { "tag": "856", "ind1": "4", "ind2": "0", "subfields": [
+      {"code": "u", "value": "http://urn.fi/URN:ISBN:978-951-9155-47-0"},
```

Comment:
Do we need to fix http to https?

Reply:
No, but it's nice to have. Also it's in the old code not affected by this PR.